### PR TITLE
zh: add example config for safe-mode

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
add example config for safe-mode

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [x] v1.0 (TiDB DM 1.0 versions)

### CODE REVIEW

**Comment by WangXiangUSTC** [comment]
@GMHDBJD PTAL

File: zh/task-configuration-file-full.md [hunk 1]
```diff
@@ -99,6 +99,7 @@ syncers:                             # syncer 处理单元运行配置参数
     worker-count: 16                 # syncer 并发同步 binlog event 的线程数量，默认值为 16
     batch: 100                       # syncer 同步到下游数据库的一个事务批次 SQL 语句数，默认值为 100
     enable-ansi-quotes: true         # 若 `session` 中设置 `sql-mode: "ANSI_QUOTES"`，则需开启此项
+    safe-mode: false                 # 设置为 true，则将来自上游的 `INSERT` 改写为 `REPLACE`，将 `UPDATE` 改写为 `DELETE` 与 `REPLACE`，保证在表结构中存在主键或唯一索引的条件下同步数据时可以重复导入 DML
```
Comment (csuzhangxc) [inline zh/task-configuration-file-full.md:102]: 也说明一下有默认 5 分钟的 safe-mode 这个行为吗？

Comment (WangXiangUSTC) [reply zh/task-configuration-file-full.md]: 已补充 f510df3

**Comment by AndreMouche** [comment]
LGTM

**Pull request merged**